spot_ros_interface/CMakeLists.txt

```diff
@@ -9,4 +9,8 @@ catkin_package()
 
 include_directories(include ${catkin_INCLUDE_DIRS})
 
-catkin_python_setup()
\ No newline at end of file
+catkin_install_python(PROGRAMS scripts/grid_utils.py
```

Comment:
spot_ros_interface uses a script-framework, therefore the scripts were added to the path. setup.py was deleted because its used to install python modules and there are no python modules in this package:

http://docs.ros.org/noetic/api/catkin/html/howto/format2/installing_python.html